### DISCUSSION TITLE
Have a Discord channel

### DISCUSSION THREAD

**Discussion by inblossoms** [mswjs/msw#1681] [Ideas]
### Scope

Adds a new behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

uh, This is it:  [Discord](https://discord.com/). 
There can be an independent community where users can have a discussion about some usage issues.

**Comment by kettanaito** [comment]
Hi, @inblossoms. We do have a Discord! MSW is a part of the [KCD Discord](https://kcd.im/discord) created and hosted by Kent C. Dodds. This is where MSW users can ask questions and get help. There are also a bunch of other channels there in general. Consider joining!

**Reply by inblossoms** [reply]
Perhaps the Discord portal could be placed on the head navigation at https://mswjs.io next to Github. 
I tried to find the header component in the document repository to make changes to this, but I couldn't find it.

**Reply by IanVS** [reply]
I joined the KCD discord, but can't find an MSW channel.  Am I just missing it?  Or, do you use one of the other channels?

**Reply by kettanaito** [reply]
@IanVS, you aren't missing anything. We used to have a designated channel there but now it's a "MSW" tag in the "#help" channel to ask questions about MSW. You can raise discussions in the general channel as well.

**Reply by kettanaito** [reply]
@inblossoms, I will think about this suggestion. We have the Discord link under the "Community" section in the footer. In the new version of the docs, I've incorporated the community links to the sidebar at the right on each documentation page so they are easier to find. That may be enough to address your concern.
